automation/schema.sql

```diff
@@ -3,6 +3,10 @@
  * IF NOT EXISTS, otherwise there will be errors
  */
 
+
+/* SCHEMA VERSION */
+/* PRAGMA schema.user_version = 2; */
```

Comment:
It wasn't, I couldn't figure out why but I didn't dig very hard. I'll remove it for now and take another stab before we want to finalize this